src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -802,14 +800,7 @@ public async ValueTask DisposeAsync()
             private async ValueTask SendIdentitiesAsync()
             {
                 //fire and forget, do not hold-up smuggler operations waiting for Raft command
-                DevelopmentHelper.ShardingToDo(DevelopmentHelper.TeamMember.Aviv, DevelopmentHelper.Severity.Normal, "try to find a better way to handle restoring identities on sharded database");
-                var databaseName = _database.Name;
-                if (_database is ShardedDocumentDatabase shardedDocumentDatabase)
-                {
-                    databaseName = shardedDocumentDatabase.ShardedDatabaseName;
-                }
-
-                await _database.ServerStore.SendToLeaderAsync(new UpdateClusterIdentityCommand(databaseName, _identities, false, RaftIdGenerator.NewId()));
+                await _database.ServerStore.SendToLeaderAsync(new UpdateClusterIdentityCommand(_database.Name, _identities, false, RaftIdGenerator.NewId()));
```

Comment:
what is `false` here?

Reply:
BTW, the comment above isn't true here... we are awaiting.

Reply:
`false` here is `force` update
removed comment